app/src/main/java/org/mozilla/fenix/library/history/viewholders/HistoryListItemViewHolder.kt

```diff
@@ -133,6 +141,21 @@ class HistoryListItemViewHolder(
                     alpha = DISABLED_BUTTON_ALPHA
                 }
             }
+            if (syncedHistoryVisible) {
+                toggleTopContent(isNormalMode)
+            }
+        }
+    }
+
+    private fun toggleTopContent(isNormalMode: Boolean) {
+        binding.syncedHistoryNavEmpty.syncedHistoryNav.run {
+            if (isNormalMode) {
+                isEnabled = true
+                alpha = 1f
+            } else {
+                isEnabled = false
+                alpha = DISABLED_BUTTON_ALPHA
+            }
```

Comment:
I would just inline this since it's confusing to see this, but also know that the `binding.recentlyClosedNavEmpty.recentlyClosedNav.run {}` block does the same thing but is not in its own function.

Reply:
I followed the lint rule
```
NestedBlockDepth:
    active: true
    threshold: 4
```